Makefile

```diff
@@ -45,33 +48,41 @@ prereqs: $(BINDIR)	## Download required utilities
 git-clone-repos:	## Clone repositories from submariner-io
 git-clone-repos:	clone-admiral clone-cloud-prepare clone-lighthouse clone-submariner
 git-clone-repos:	clone-submariner-operator clone-shipyard
+	@echo -- $@ --
 
 clone-%: %/.git
+	@echo -- $@ --
 	@echo -n
 
 .SECONDARY: %/.git
 %/.git: 
+	@echo -- $@ --
 	@if [ ! -d  $@ ]; then (git clone $(SUBMARINER_IO_GH)/$*.git); fi
 	@(cd $*; git remote rename origin submariner)
 	@(cd $*; git remote add axon $(AXON_NET_GH)/$*.git)
 
 git-fetch-latest:	## Fetch latest repositories from upstream, does *not* rebase
 git-fetch-latest: fetch-latest-admiral fetch-latest-cloud-prepare fetch-latest-lighthouse
 git-fetch-latest: fetch-latest-submariner fetch-latest-submariner-operator fetch-latest-shipyard
+	@echo -- $@ --
 
 fetch-latest-%: clone-%
 	@echo -- $@ --
 	@(cd $*; git fetch submariner devel)
 
-remove-git-repos:	## Remove local copy of upstream repositories
-remove-git-repos: remove-admiral remove-cloud-prepare remove-lighthouse remove-submariner
-remove-git-repos: remove-submariner-operator remove-shipyard
+git-stable:	## Update repositories to last set tag
```

Comment:
I *think* the message was changed since we're no longer taking the last tag but a user defined tag.
Please ensure you're rebased on the latest.